docs/vc-http-api.yml

```diff
@@ -89,6 +89,30 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /credentials/verifyMany:
+    post:
+      tags:
+        - Verifier
+      summary: Verifies a list of verifiableCredentials and returns a list of index matched verificationResults in the response body.
+      operationId: verifyCredentials
+      description: Verifies a list of verifiableCredentials and returns a list of index matched verificationResults in the response body.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/VerifyCredentialsRequest"
+        description: Parameters for verifying a list of verifiableCredentials.
+      responses:
+        "200":
+          description: Verifiable Credentials successfully verified!
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VerifyCredentialsResponse"
+        "400":
+          description: invalid input!
```

Comment:
Needs a more thorough description.

Reply:
matched existing API as before.  Added a bit of clarity there